GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -552,7 +552,7 @@ private async Task ShowSelectedFileBlameAsync(bool ensureNoSwitchToFilter, int?
         /// Activate diffviewer if Blame is visible
         /// </summary>
         /// <returns>a task</returns>
-        private async Task ShowSelectedFileDiffAsync(bool ensureNoSwitchToFilter = false)
+        private async Task ShowSelectedFileDiffAsync(bool ensureNoSwitchToFilter, int? line)
```

Comment:
This does feel wrong to do `int?`.... Maybe test for negativity, e.g., `if (line >= 0)`?

Reply:
-1 can be used instead of null in the meaning as "there is no line to go to".
This will affect a lot of existing code for blame etc though to avoid to have two implementation so I rather keep nullable.